### ISSUE TITLE
Some bug maybe?

### ISSUE THREAD

**Issue by ddrise** [xiangzhang1015/EEG-based-Control#1]
hey, 
In EEG_based_control_train.py, line 57, 
np.random.shuffle(all)  # mix eeg_all

I think you should not shuffle the eeg data, since you will use lstm.

waiting for your reply

**Comment by rugolotti** [comment]
I agree with @ddrise. It is a bug. If you remove it, the model does not learn. In my opinion, the reason for it to work is that if you first shuffle and then split between training and test, there is a very good chance to have example both in training and test with the last input with extremely high correlation. Maybe the network learns to give an answer based only on the last tick, ignoring what is preceding it?

**Comment by ddrise** [comment]
@rugolotti Yes. Besides, 1 layer LSTM is not stronger than 1 layer FC net. So I guess the accuracy comes from the correlation entirely.......
